content/rs/administering/creating-databases/create-active-active.md

```diff
@@ -110,7 +109,12 @@ Every instance of an Active-Active database can receive write operations, and al
         can use [Multi-key commands]({{< relref "/rs/concepts/high-availability/clustering.md" >}})
         without the limitations.
 
-    - **Eviction policy** - The eviction policy for Active-Active databases is `noeviction`.
+    - [**OSS Cluster API**]({{< relref "/rs/administering/designing-production/networking/using-oss-cluster-api.md" >}}) - {{< embed-md "oss-cluster-api-intro.md"  >}}
+
+    - [**Data eviction policy**]({{< relref "/rs/administering/database-operations/eviction-policy.md" >}}) -
+        By default, when the total size of the database reaches its memory limit the database evicts keys
```

Comment:
By default for A/A there is no eviction policy :) 
@bmansheim

Reply:
So what happens when the limit is reached? Error?